Examples/Kitura/Package.resolved

```diff
@@ -6,116 +6,107 @@
         "repositoryURL": "https://github.com/IBM-Swift/BlueSocket.git",
         "state": {
           "branch": null,
-          "revision": "ce36f72f2c36b88f847de50c8c0ed8021ff80097",
-          "version": "1.0.16"
+          "revision": "ccc671b022bc60e5fffcb2aa15f499c6331bfd18",
```

Comment:
should we even bother keeping the package.resolveds around in these samples?

Reply:
I'm down to remove them. The package lock files for node are this sort of endless PITA because there are always conflicts. I don't think we have that problem here, but I'm not attached to them.

Reply:
I think they're possibly useful for the driver so we can coordinate on exactly what version we've been using, but since these example projects are pretty stagnant and basic it doesn't really add much